config/hartrao_reduced_links.yaml

```diff
@@ -0,0 +1,333 @@
+      lost_samples_buf: lost_samples_buffer_2
+    - dpdk_handler: iceBoardStandard
+      out_buf: gpu_input_buffer_3
+      lost_samples_buf: lost_samples_buffer_3
+    - dpdk_handler: iceBoardStandard
+      out_buf: gpu_input_buffer_4
+      lost_samples_buf: lost_samples_buffer_4
+    - dpdk_handler: iceBoardStandard
+      out_buf: gpu_input_buffer_5
+      lost_samples_buf: lost_samples_buffer_5
+    - dpdk_handler: none
+    - dpdk_handler: none
+#    - dpdk_handler: iceBoardStandard
+#      out_buf: gpu_input_buffer_6
+#      lost_samples_buf: lost_samples_buffer_6
+#    - dpdk_handler: iceBoardStandard
+#      out_buf: gpu_input_buffer_7
+#      lost_samples_buf: lost_samples_buffer_7
+
+zero_samples:
+  zero_0:
+    kotekan_stage: zeroSamples
+    out_buf: gpu_input_buffer_0
+    lost_samples_buf: lost_samples_buffer_0
+  zero_1:
+    kotekan_stage: zeroSamples
+    out_buf: gpu_input_buffer_1
+    lost_samples_buf: lost_samples_buffer_1
+  zero_2:
+    kotekan_stage: zeroSamples
+    out_buf: gpu_input_buffer_2
+    lost_samples_buf: lost_samples_buffer_2
+  zero_3:
+    kotekan_stage: zeroSamples
+    out_buf: gpu_input_buffer_3
+    lost_samples_buf: lost_samples_buffer_3
+  zero_4:
+    kotekan_stage: zeroSamples
+    out_buf: gpu_input_buffer_4
+    lost_samples_buf: lost_samples_buffer_4
+  zero_5:
+    kotekan_stage: zeroSamples
+    out_buf: gpu_input_buffer_5
+    lost_samples_buf: lost_samples_buffer_5
+#  zero_6:
+#    kotekan_stage: zeroSamples
+#    out_buf: gpu_input_buffer_6
+#    lost_samples_buf: lost_samples_buffer_6
+#  zero_7:
+#    kotekan_stage: zeroSamples
+#    out_buf: gpu_input_buffer_7
+#    lost_samples_buf: lost_samples_buffer_7
+
+merge:
+  kotekan_stage: bufferMerge
+  in_bufs:
+    - gpu_input_buffer_0
+    - gpu_input_buffer_1
+    - gpu_input_buffer_2
+    - gpu_input_buffer_3
+    - gpu_input_buffer_4
+    - gpu_input_buffer_5
+#    - gpu_input_buffer_6
+#    - gpu_input_buffer_7
+  out_buf: merged_network_buf
+
+gpu:
+  num_data_sets: 1
+  profiling: true
+  legacy_opencl: true
+  kernel_path: "/var/lib/kotekan/opencl/kernels/"
+  log_level: WARN
+  frame_arrival_period: samples_per_data_set / 390625 / num_links
+  commands: &command_list
+    - name: clInputData
+    - name: clPresumZero
+    - name: clOutputDataZero
+    - name: clPresumKernel
+    - name: clKVCorr
+    - name: clOutputData
+  gpu_0:
+    kotekan_stage: clProcess
+    gpu_id: 0
+    commands: *command_list
+    in_buffers:
+      network_buf: merged_network_buf
+    out_buffers:
+      output_buf: gpu_output_buffer
+
+split:
+  kotekan_stage: BufferSplit
+  in_buf: gpu_output_buffer
+  out_bufs:
+    - gpu_output_split_buffer_0
+    - gpu_output_split_buffer_1
+    - gpu_output_split_buffer_2
+    - gpu_output_split_buffer_3
+    - gpu_output_split_buffer_4
+    - gpu_output_split_buffer_5
+#    - gpu_output_split_buffer_6
+#    - gpu_output_split_buffer_7
+
+metadata:
+  #kotekan_stage: chimeMetadataDump
```

Comment:
Is this debug?

Reply:
Oh yeah I didn't even notice that line was there.